modules/local/align/bwamem2.nf

```diff
@@ -0,0 +1,55 @@
+process ALIGN_BWAMEM2 {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "${moduleDir}/environment.yml"
```

Comment:
there isn't such a file yet and probably wouldn't work with conda, so remove or comment out this line for now?

Reply:
Line removed. Thanks!